csrc/kernel_cache.cpp

```diff
@@ -851,13 +851,21 @@ void FusionKernelRuntime::compileFusionParallel(KernelArgumentHolder args) {
       group_runtime_inputs.push(args_manager.checkTensorMap(input));
     }
 
-    // launch compileKernel thread here
-    getThreadPool()->run([=]() {
-      FUSER_PERF_SCOPE("FusionKernelRuntime::compileFusionParallel");
+    bool serial_compile = true;
+    if (serial_compile) {
```

Comment:
Why the change here?

Reply:
Ah. I forgot to reset this! I wind up doing this sometimes for easier debugging of compile failures. I'll fix this, and maybe add a small PR in the future that disables multithreaded compilation with an `NVFUSER_DISABLE` option.